src/index.ts

```diff
@@ -151,6 +217,47 @@ export class CheckoutInstance {
     }
   }
 
+  /**
+   * Creates popup modal in the DOM to host the Checkout iframe.
+   *
+   * If the Checkout becomes completed or if the user clicks cancelled, automatically closes the popup.
```

Comment:
How does the dialog close automatically?

Reply:
done.